### ISSUE TITLE
Sync names

### ISSUE THREAD

**Issue by ale5000-git** [microg/IchnaeaNlpBackend#34]
Hi,
on GitHub the name is IchnaeaNlpBackend, on F-Droid the name is MozillaNlpBackend.

I think it would be better to rename one of them to reduce confusion.

**Comment by rugk** [comment]
I think for the end user "Ichnaea" is a very problematic name. It is hard to pronounce and it is not obvious this has a connection to Mozilla. BTW, what dopes the name even mean?

**Comment by realmain** [comment]
@rugk it means the tracker, from Greek mythology.

**Comment by breversa** [comment]
I agree that a rename is welcome, but @rugk is right : the link with Mozilla is totally absent if we go with Ichnaea.
